controller/iteration_blackbox_test.go

```diff
@@ -593,6 +595,77 @@ func (rest *TestIterationREST) TestRootIterationCanNotStart() {
 	test.UpdateIterationBadRequest(rest.T(), svc.Context, svc, ctrl, ri.ID.String(), &payload)
 }
 
+func (rest *TestIterationREST) createIterations() (*app.IterationSingle, *account.Identity) {
+	sp, _, _, _, parent := createSpaceAndRootAreaAndIterations(rest.T(), rest.db)
+	_, err := rest.db.Iterations().Root(context.Background(), parent.SpaceID)
+	require.Nil(rest.T(), err)
+	parentID := parent.ID
+	name := uuid.NewV4().String()
```

Comment:
Please use `name := testsupport.CreateRandomValidTestName("foo-")` instead of `name := uuid.NewV4().String()`. It creates valid random names.

Reply:
Thanks for pointing out

Reply:
@kwk handled it here 89e2f1bee73eb8f0d602cfeb75d65b75d23a2828